auth0_flutter_platform_interface/lib/src/web-auth/web_auth_login_options.dart

```diff
@@ -1,40 +1,33 @@
-import '../request/request_options.dart';
-import 'id_token_validation_config.dart';
+import '../id_token_validation_config.dart';
+import '../login_options.dart';
 
-class WebAuthLoginOptions implements RequestOptions {
-  final IdTokenValidationConfig? idTokenValidationConfig;
-  final String? audience;
-  final Set<String> scopes;
-  final String? redirectUrl;
-  final String? organizationId;
-  final String? invitationUrl;
+class WebAuthLoginOptions extends LoginOptions {
   final bool useEphemeralSession;
-  final Map<String, String> parameters;
   final String? scheme;
 
   WebAuthLoginOptions(
-      {this.idTokenValidationConfig,
-      this.audience,
-      this.scopes = const {},
-      this.redirectUrl,
-      this.organizationId,
-      this.invitationUrl,
+      {final IdTokenValidationConfig? idTokenValidationConfig,
```

Comment:
I think these can be defined using `super`:

<img width="360" alt="Screenshot 2023-02-23 at 16 16 02" src="https://user-images.githubusercontent.com/5055789/221008101-5d48e5ed-5334-4a99-8f4b-86b577fb9730.png">

Reply:
Yes, my bad 🤦🏼